pkg/artifact/ingest_logs.go

```diff
@@ -121,7 +133,9 @@ func (f *logIngester) ProcessElement(ctx context.Context, event EventRecord) Art
 		"delivery_id", event.DeliveryID,
 		"event", event,
 		"result", result)
-	if err := f.handleMessage(ctx, event.RepositoryName, event.LogsURL, gcsPath); err != nil {
+
+	err := f.handleMessage(ctx, event.LogsURL, gcsPath)
+	if err != nil {
```

Comment:
nit: prefer single line unless extremely long

```suggestion
	if err := f.handleMessage(ctx, event.LogsURL, gcsPath); err != nil {
```

Reply:
Done